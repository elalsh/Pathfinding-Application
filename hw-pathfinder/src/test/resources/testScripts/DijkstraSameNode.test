# Find a path with Dijkstra's algorithm when
# src and dest node are the same.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A B 1.0
AddEdge simpleGraph A A 50.0

FindPath simpleGraph A A