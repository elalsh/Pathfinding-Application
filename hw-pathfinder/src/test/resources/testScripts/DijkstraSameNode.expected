# Find a path with Dijkstra's algorithm when
# src and dest node are the same.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to B in simpleGraph
added edge 50.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000